packages/components/src/IllustratedMessage/docs/buttons.tsx

```diff
@@ -0,0 +1,25 @@
+import { Button, ButtonGroup, Content, Heading, IllustratedMessage, Stack, SvgImage } from "@hopper-ui/components";
+
+import { NoResults } from "../assets/index.ts";
+
+export default function Example() {
+    return (
+        <Stack>
```

Comment:
can you add a small gap here, they are too close to each other

Reply:
In fact never, can you create 2 different sample, and put them back to back

Reply:
like in this section https://wl-orbiter-website.netlify.app/?path=/docs/card--docs

under buttons part